sphinx/application.py

```diff
@@ -117,8 +116,7 @@ def __init__(self, srcdir, confdir, outdir, doctreedir, buildername,
                  parallel=0):
         # type: (unicode, unicode, unicode, unicode, unicode, Dict, IO, IO, bool, bool, List[unicode], int, int) -> None  # NOQA
         self.verbosity = verbosity
-        self._extensions = {}                   # type: Dict[unicode, Any]
-        self._extension_metadata = {}           # type: Dict[unicode, Dict[unicode, Any]]
```

Comment:
If you means "Although deleting this may have an effect, I think that it does not have to worry because it starts with an underscore.", I give +1 ;)